v2_plugin.yaml

```diff
@@ -362,11 +408,15 @@ relationships:
         unlink:
           implementation: ansible.cloudify_ansible.tasks.cleanup
 
-  cloudify.ansible.relationships.connected_to_host:
-    derived_from: cloudify.relationships.ansible.connected_to_host
-
-  cloudify.ansible.relationships.run_on_host:
-    derived_from: cloudify.relationships.ansible.run_on_host
```

Comment:
We need to keep the old relationships, because there are existing blueprints that use them. I know it's a major plugin upgrade, but we haven't deprecated them formally.